repos/system_upgrade/common/actors/cloud/checkhybridimage/actor.py

```diff
@@ -8,14 +8,9 @@
 class CheckHybridImage(Actor):
```

Comment:
Sorry, naming - maybe let's add `Azure` to the name if the actor is not to be generic?

Reply:
I agree that the names are very generic. However the name I really don't like is `CheckGrubenvToFile`, which I understand has been there before, but it's not very descriptive.

Reply:
hmm...maybe `CheckGrubenbSymlink` ?

Reply:
I have changed the name to ``CheckHybridImage``, but I would like to keep ``CheckGrubenvToFile`` since it is so tightly coupled with ``ConvertGrubenvToFile`` @pirat89 @matejmatuska